src/components/AddAccountDialog.js

```diff
@@ -7,7 +7,7 @@ import TextField from '@material-ui/core/TextField';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
 import FormGroup from '@material-ui/core/FormGroup';
 import Switch from '@material-ui/core/Switch';
-import { Account } from '@solana/web3.js';
+import { decodeAccount } from './helpers/import';
```

Comment:
I would put this in `src/utils`, since that is the convention of the project

Reply:
Thanks I made this change.